apps/vr-tests-react-components/src/stories/Toolbar/Toolbar.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Toolbar, ToolbarProps, ToolbarButton, ToolbarDivider, ToolbarToggleButton } from '@fluentui/react-toolbar';
+import { Menu, MenuTrigger, MenuPopover, MenuList, MenuItem } from '@fluentui/react-menu';
+import {
+  TextBold24Regular,
+  TextItalic24Regular,
+  TextUnderline24Regular,
+  AlertSnooze24Regular,
+  MoreHorizontal24Filled,
+} from '@fluentui/react-icons';
+import { ComponentMeta } from '@storybook/react';
+import { withScreenerSteps } from '../../utilities';
+
+export default {
+  title: 'Toolbar Converged',
+  Component: Toolbar,
+  decorators: [story => withScreenerSteps({ story, steps: [] })],
+} as ComponentMeta<typeof Toolbar>;
+
+export const Default = (props: Partial<ToolbarProps>) => (
+  <Toolbar {...props}>
+    <ToolbarButton aria-label="Text option - Bold" appearance="primary" icon={<TextBold24Regular />} />
```

Comment:
I would test the toggle button toggle state too, it's probably one of the few 'design' differences between the standard toggle button